py/htm/examples/rest/client.py

```diff
@@ -53,20 +53,11 @@ def main(argv):
        {addLink:   {src: "encoder.encoded", dest: "sp.bottomUpIn"}},
        {addLink:   {src: "sp.bottomUpOut", dest: "tm.bottomUpIn"}}
     ]}"""
+
+  net = rest_api.Network(config, host = URL, verbose=verbose)
     
   # Send the config string and obtain a token for the created Network object instance.
-  if verbose: print('Post /network; body: '+config)
-  res = requests.post(URL+'/network', data=config)
```

Comment:
I like the simplification this provides, but I don't know if the example shouldn't be explicit on how the REST is being used here

Reply:
..depends on our stance below https://github.com/htm-community/htm.core/pull/834#discussion_r437267218